Minor fixes + some slight styling of cards

jmorg opened the pull request:
- Change how we flatten Objects into GeoJSON
- Start styling popup cards
- Add some additional information from mock data to popups
- Tuck some additional data directly into JS instead of requiring a server
- Update some code formatting for consistency

jmorg committed:
Formatting fix.

jmorg committed:
Import us-states-id_albers as a var instead of getting it from the server

jmorg committed:
Make custom flattening function. .Flat was throwing errors which was causing sample data to not load. This is a bit cumbersome, but the mapping functions we use return an Object, not an array, and we need to reformat it to make it valid GeoJSON.

jmorg committed:
Minor formatting changes.

jmorg committed:
Minor styling of pop up cards.

patrickvossler18 commented on assets/js/main_albers.js:
```diff
@@ -612,22 +613,37 @@ map.on('load', function() {
         data: {
                 'type': 'FeatureCollection',
-                'features': district_data.map(district => getDistrictShapes(district, true)).flat()
+                'features': flattenDistrictDataIntoGeoJSON(
```
It seems like these produce the same output of [featureObject, featureObject], e.g. 
![image](https://user-images.githubusercontent.com/938890/88099894-abe5ae80-cb50-11ea-80f8-cf6e7734177a.png).

Is this method better because it's more general?

patrickvossler18 committed:
Remove json object from the array

Per the documentation (https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson-data), it seems mapbox doesn't like geojson in an array as an argument for data
